src/client/play/js/events.js

```diff
@@ -12,6 +12,21 @@ import Tile from './tile';
 import tiles from './tiles';
 import ui from './ui';
 
+const KEY_UP = "KeyW";
+const KEY_DOWN = "KeyS";
+const KEY_LEFT = "KeyA";
+const KEY_RIGHT = "KeyD";
+
+const KEY_ZOOM_OUT = "KeyQ";
+const KEY_ZOOM_IN = "KeyE";
+
+const KEY_EXPLORE = "KeyC";
+const KEY_ROTATE_TILE_CLOCKWISE = "KeyR";
+const KEY_ROTATE_TILE_COUNTER_CLOCKWISE = "KeyT";
+const KEY_CANCEP = "Escape";
```

Comment:
Watch out, I think you have a typo here and when referencing this variable (`KEY_CANCEL` not `KEY_CANCEP`?)

Reply:
> Watch out, I think you have a typo here and when referencing this variable (`KEY_CANCEL` not `KEY_CANCEP`?)

Thanks for catching this! Fixed and pushed!

Reply:
Awesome!